it/cs-229-probability.md

```diff
@@ -0,0 +1,385 @@
+
+&#10230; Disuguaglianza di Chebyshev ― Sia X una variabile aleatoria con valore atteso μ. Per k,σ>0 abbiamo la seguente disuaglianza:
+
+<br>
+
+**34. Main distributions ― Here are the main distributions to have in mind:**
+
+&#10230; Distribuzioni principali ― Di seguito le distribuzioni principali da tenere a mente:
+
+<br>
+
+**35. [Type, Distribution]**
+
+&#10230; [Tipologia, Distrubuzione]
+
+<br>
+
+**36. Jointly Distributed Random Variables**
+
+&#10230; Distribuzione congiunta di variabili aleatorie
+
+<br>
+
+**37. Marginal density and cumulative distribution ― From the joint density probability function fXY , we have**
+
+&#10230; Densità marginale e distribuzione cumulativa ― Dalla funzione di densità congiunta fXY abbiamo che:
+
+<br>
+
+**38. [Case, Marginal density, Cumulative function]**
+
+&#10230; [Caso, Densità marginale, Funzione cumulativa]
+
+<br>
+
+**39. Conditional density ― The conditional density of X with respect to Y, often noted fX|Y, is defined as follows:**
+
+&#10230; Densità condizionata ― La densità condizionata di X rispetto a Y, spesso denotata come fX|Y, è definita come:
+
+<br>
+
+**40. Independence ― Two random variables X and Y are said to be independent if we have:**
+
+&#10230; Indipendenza ― Due variabili aleatorie X e Y si dicono indipendenti se:
+
+<br>
+
+**41. Covariance ― We define the covariance of two random variables X and Y, that we note σ2XY or more commonly Cov(X,Y), as follows:**
+
+&#10230; Covarianza ― Si definisce la covarianza di due variabili aleatorie X e Y, denotata da σ2XY o più comunemente Cov(X,Y), come segue:
+
+<br>
+
+**42. Correlation ― By noting σX,σY the standard deviations of X and Y, we define the correlation between the random variables X and Y, noted ρXY, as follows:**
+
+&#10230; Correlazione ― Date σX,σY le deviazioni standard di X e Y, definiamo la correlazione tra le variabili aleatorie X e Y, denotata da ρXY, come segue:
+
+<br>
+
+**43. Remark 1: we note that for any random variables X,Y, we have ρXY∈[−1,1].**
+
+&#10230; Osservazione 1: notiamo che per ogni variabile aleatoria X,Y, abbiamo che ρXY∈[−1,1].
+
+<br>
+
+**44. Remark 2: If X and Y are independent, then ρXY=0.**
+
+&#10230; Osservazione 2: Se X e Y sono indipendenti, allora ρXY=0.
+
+<br>
+
+**45. Parameter estimation**
+
+&#10230; Stima dei parametri
+
+<br>
+
+**46. Definitions**
+
+&#10230; Definizioni
+
+<br>
+
+**47. Random sample ― A random sample is a collection of n random variables X1,...,Xn that are independent and identically distributed with X.**
+
+&#10230; Campione casuale ― Un campione casuale è un gruppo di n variabili aleatorie X1,...,Xn distribuite in modo indipendente e indentico con X.
+
+<br>
+
+**48. Estimator ― An estimator is a function of the data that is used to infer the value of an unknown parameter in a statistical model.**
+
+&#10230; Stimatore ― Uno stimatore è una funzione dei dati usata per dedurre il valore di un parametro sconosciuto in un  modello statistico.
```

Comment:
&#10230; Stimatore ― Uno stimatore è una funzione dei dati usata per dedurre il valore di un parametro sconosciuto in un modello statistico.

Reply:
double *space* character between "un" and "modello"
The only error/typo I found